### PR TITLE
Add NEWS for 0.26

### PR DESCRIPTION
**Pull request by timholy**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 For the full changes, please check the git history and the [release page](https://github.com/JuliaImages/Images.jl/releases)
 
+# v0.26
```
Comment (ashwani-rathee) [inline NEWS.md:5]: This is looking good, let's release Images.jl 0.26: hopefully close #1026 , #1013, #841 , #682. But that, we need to deprecate these: https://github.com/JuliaImages/Images.jl/blob/4854014dd408969a640f03e20fd8d74f350607fb/src/algorithms.jl#L137. These two weren't on my mind just noticed. Also I am not sure if we need these still: `test_approx_eq_sigma_eps`

Comment (ashwani-rathee) [reply NEWS.md:5]: Equivalent functions are in HistogramThresholding.jl though and is rexported by ImageBinarization.jl which we are adding in this update.

Comment (timholy) [reply NEWS.md:5]: Can you make this fix? I think you know that package better than I do.

Comment (ashwani-rathee) [reply NEWS.md:5]: Sure thing, let me give it a shot!

Comment (timholy) [reply NEWS.md:5]: NVM, I'll give it a whirl unless you're already working on it; I'm sure you're busy with the presentation...

Comment (timholy) [reply NEWS.md:5]: Hah, one thing we hadn't done was remove the old deprecations!

Comment (ashwani-rathee) [reply NEWS.md:5]: Yeah, I am thinking of wrapping mostly today only

Comment (timholy) [reply NEWS.md:5]: I'm already working on the fix now. It turns out to be more compllicated: we have to delete the entire contents of `src/deprecations.jl` (they're all older than 2 years, and depwarns should last only until the next breaking release) and put the new deprecations there.

Comment (timholy) [reply NEWS.md:5]: xref https://github.com/JuliaImages/ImageContrastAdjustment.jl/pull/59

**Pull request merged**